s4e-web/package.json

```diff
@@ -3,11 +3,19 @@
   "version": "0.0.2",
   "scripts": {
     "ng": "ng",
+    "start-notifications": "ng build notifications --watch",
+    "build-libraries": "ng build notifications",
+    "prestart": "node_modules/.bin/npm run build-libraries",
```

Comment:
Does it have to be repeated so many times? Isn't there some way to automatically run the build-libraries when running/building the app?

Reply:
unfortunately there is no way (at least I could not found one) that was one of the "issues" of using library which I mentioned. They are not included in the standard `ng build`

Reply:
Understood, thanks for clarification